python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_adapter_config.schema.yml

```
@@ -135,11 +135,13 @@ $defs:
             default: false
           endpoint_role:
             type: str
+            description: Set `dynamic` instead of `bmca`(deprecated).
             default: "follower"
             valid_values:
               - "bmca"
               - "default"
               - "follower"
+              - "dynamic"
```

Let's reorder so default is first and deprecated is last.
```suggestion
              - "follower"
              - "dynamic"
              - "bmca"
              - "default"
```

---

Done.